### DISCUSSION TITLE
zoomeye api can't use[Issue]

### DISCUSSION THREAD

**Discussion by DaYuanzhang** [projectdiscovery/subfinder#532] [Q&A]
zoomeye api can't use

**Comment by ehsandeep** [comment]
@DaYuanzhang do you have more details to share? what is not working and what you have tried or any error details?

**Comment by DaYuanzhang** [comment]
![image](https://user-images.githubusercontent.com/67625626/161875534-f5fd3f22-60e9-4382-ad5b-646f87efc5b0.png)
![image](https://user-images.githubusercontent.com/67625626/161875656-1c51ab6d-6286-4472-aa96-a9e431612533.png)
![image](https://user-images.githubusercontent.com/67625626/161875709-bebc6df4-505a-4ace-bee2-2913ce518c02.png)
![image](https://user-images.githubusercontent.com/67625626/161875743-79070dcc-6354-471b-8a66-9f987a14c0cf.png)

**Comment by ehsandeep** [comment]
@DaYuanzhang there is 2 version of zoomeye source:

1. zoomeye ( works with login + password)
2. zoomeyeapi (works with API)

You need to configure the key for `zoomeyeapi` and use the same source with `-s` flag.

**Reply by DaYuanzhang** [reply]
but my config is right
![image](https://user-images.githubusercontent.com/67625626/162273540-509b7801-b908-47dd-bba7-3ba53f0161a8.png)
![image](https://user-images.githubusercontent.com/67625626/162273567-d30a40d6-8791-41a1-b821-58fd29ff1d9b.png)
Is that something wrong?
